mmv1/products/accesscontextmanager/EgressPolicy.yaml

```diff
@@ -0,0 +1,64 @@
+# Copyright 2018 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+--- !ruby/object:Api::Resource
+  name: 'EgressPolicy'
+  create_url: "{{egress_policy_name}}"
+  base_url: ""
+  self_link: "{{egress_policy_name}}"
+  create_verb: :PATCH
+  delete_verb: :PATCH
+  immutable: true
+  update_mask: true
+  identity:
+    - resource
+  nested_query: !ruby/object:Api::Resource::NestedQuery
+    modify_by_patch: true
+    is_list_of_ids: true
+    keys:
+      - status
+      - resources
+  references: !ruby/object:Api::Resource::ReferenceLinks
+    api: 'https://cloud.google.com/access-context-manager/docs/reference/rest/v1/accessPolicies.servicePerimeters#egresspolicy'
+  description: |
+    EgressPolicies match requests based on egressFrom and egressTo stanzas. 
+    For an EgressPolicy to match, both egressFrom and egressTo stanzas must be matched. 
+    If an EgressPolicy matches a request, the request is allowed to span the ServicePerimeter 
+    boundary. For example, an EgressPolicy can be used to allow VMs on networks 
+    within the ServicePerimeter to access a defined set of projects outside the 
+    perimeter in certain contexts (e.g. to read data from a Cloud Storage bucket 
+    or query against a BigQuery dataset).
+  autogen_async: true
+  exclude_validator: true
+  # Skipping the sweeper due to the non-standard base_url and because this is fine-grained under ServicePerimeter/IngressPolicy
+  skip_sweeper: true
+  id_format: "{{egress_policy_name}}/{{resource}}"
+  import_format: ["{{egress_policy_name}}/{{resource}}"]
+  custom_code: !ruby/object:Provider::Terraform::CustomCode
+    custom_import: templates/terraform/custom_import/access_context_manager_service_perimeter_egress_policy.go.erb
+  parameters:
+    - !ruby/object:Api::Type::ResourceRef
+      name: 'egressPolicyName'
```

Comment:
I would expect the name of this field to be something like `service_perimeter` since it is a reference to a service perimeter, not to an egress policy. (Since this resource is the egress policy.)

However, it looks like this would match the ingress policy resource added last week, so I'm fine with moving forward (unless we want to try to modify both.)

Reply:
I'd prefer keeping it as it is to match ingress_policy

Reply:
Since we have the opportunity, it would be great to modify both this & ingress policy to use `service_perimeter` (or some other name that refers to the target instead of the current resource) instead.

Reply:
but I leave the decision to you.